### PR TITLE
Trigger when click in 'retry' for image snapshot

### PR DESCRIPTION
**Pull request by vendramini**
In my interface, when user retry the snapshot I must disable the `next` button. The user must take a picture to continue.

I didn't find any callback for this, so I implemented one.

I choose `replay` for event name because is the same name of css class.

### CODE REVIEW

**Comment by thijstriemstra** [comment]
@vendramini thanks. What version of video.js are you using?

**Comment by vendramini** [comment]
```javascript
"video.js": "^7.6.0",
"videojs-record": "^3.8.0"
```
Why do travis failed? I just cloned your repo and added 2 lines of code =/
Does this PR makes sense to you? Could I get this callback in some other way without this PR?

**Comment by thijstriemstra** [comment]
> Why do travis failed? 

I still haven't figured out why, you can ignore it (appveyor does not fail for example).

> // trigger replay callback

Would it make sense to add this to the other modes as well? e.g. audio, audio/video, etc

**Comment by vendramini** [comment]
> Would it make sense to add this to the other modes as well? e.g. audio, audio/video, etc
I don't think so, because the replay function on audio/video just play the content again, it's already recorded and stored. If user wants to record again, they should click on `Record` (circle button) again, and its starts recording again, so we always have recorded content. 1 step here.

In case of picture don't: user take a picture and if retry/replay is pressed we will have another picture just if they click on camera icon again. 2 steps here.

Does it makes sense? If you want I could add on audio/video too.

File: src/js/controls/camera-button.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ class CameraButton extends Button {
 
             // reset camera button
             this.onStop();
+
+            // trigger replay callback
```
Comment (thijstriemstra) [inline src/js/controls/camera-button.js]: i think we should describe it as 'event' instead of 'callback'..

**Comment by vendramini** [comment]
Sure! As we will not add this event on audio/video, I think rename it from `Event.REPLAY` to `Event.RETRY` specifying just for picture case makes more sense. What do you think?

**Comment by thijstriemstra** [comment]
sounds good @vendramini

**Comment by vendramini** [comment]
Done.

@thijstriemstra after taking picture, I would like to get `blob` from canvas, not `base64`. I didn't find a way to return blob from your lib, so I'm doing this:
```javascript
const divCanvas: HTMLDivElement = document.getElementsByClassName('vjs-record-canvas');
        const canvas: HTMLCanvasElement = divCanvas[0].firstChild;
        canvas.toBlob(/*callback*/);
```
Would a PR to choose between base64 or blob (in photos) be interesting? I could do, if you want.

**Comment by thijstriemstra** [comment]
>Would a PR to choose between base64 or blob (in photos) be interesting? I could do, if you want.

@vendramini sounds good, thanks.

**Pull request merged**